core/utils/src/rootcling.cxx

```diff
@@ -4153,10 +4156,14 @@ int RootCling(int argc,
    }
 
    std::string resourceDir;
-#ifdef R__LLVMRESOURCEDIR
-   resourceDir = "@R__LLVMRESOURCEDIR@";
+#ifdef R__EXTERN_LLVMDIR
+   resourceDir = R__EXTERN_LLVMDIR;
```

Comment:
If I understand correctly, `@R__LLVMRESOURCEDIR@` was used to re-write the source by CMake. Given that we don't need this anymore, should the CMake re-writing also go away? (Let me know if this is not that obvious to find and I'll ask our CMake specialist who implemented this.)

I also don't understand the change completely - R__EXTERN_LLVMDIR is for a patches llvm tree somewhere outside ROOT, while R__LLVMRESOURCEDIR is the location of the build's resource dir. Aren't those two completely different?